container/container.go

```diff
@@ -0,0 +1,57 @@
+package container
+
+import (
+	"context"
+	"errors"
+
+	godocker "github.com/fsouza/go-dockerclient"
+)
+
+// FindContainerStrict returns a docker container if exist. Return error if not found.
+func FindContainerStrict(namespace []string) (container *godocker.APIContainers, err error) {
```

Comment:
What do you think about this function Strict? the logic is to return an error if the container is nil.

Reply:
Ok good, I also prefer to use the strict way.